### PR TITLE
Feature/add jonas aschenbrenner

### LINKED ISSUE #59: Add Jonas Aschenbrenner to website

### PR DESCRIPTION
**Pull request by Sanjo**
Fixes #59 

This needs to be merged after #64 

Open question at @hellerve: Regarding the [job title](https://github.com/port-zero/handbook/blob/master/job_titles.md) am I supposed to replace the "Speciality" in "Speciality Senior Engineers" with something more specific?

### CODE REVIEW

**Comment by hellerve** [comment]
You can leave that part of your title altogether! These differences are purely for internal purposes, and not really relevant for the website. I do appreciate you asking, though.

And in general, we’d usually just take the title as-is, and explain what your specialty is if necessary!

File: content/team/jonas-aschenbrenner.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+description: Speciality Senior Engineer
+weight: -5
+type: teammember
+image: img/team/jonas_aschenbrenner.jpg
+---
+
+Jonas was techical co-founder of multiple start-ups.
+He has over 10 years of professional experience in the area
+of software development, specifically in the area of web development.
+He also also has highly developed communication skills.
```
Comment (hellerve) [inline content/team/jonas-aschenbrenner.md]: There’s a duplicate also here :)

Comment (Sanjo) [reply content/team/jonas-aschenbrenner.md]: Thanks. :-)

**Comment by hellerve** [comment]
If you rebase this as well and change your title from “Speciality Senior Engineer” to ”Senior Engineer” we can merge this as well! :+1:

**Comment by hellerve** [comment]
Do you know why we still see all of the stale commits? Is it possibly because of the rebase?

**Comment by hellerve** [comment]
Perfect, thank you!

**Comment by Sanjo** [comment]
> Do you know why we still see all of the stale commits? Is it possibly because of the rebase?

I have now merged master into this PR and I have changed the job title to ”Senior Engineer”. Now the list of changes in this PR is also clean.

__Side note:__ I don't do rebases for pushed branches. I do no-ff merges instead to avoid situations where pulls are broken for other people who had pulled the branch before.

**Comment by hellerve** [comment]
Perfect!

**Pull request merged**